test/cppsuite/test_harness/thread_context.h

```diff
@@ -6,7 +6,9 @@
 #include "wiredtiger.h"
 
 namespace test_harness {
+/* Define the different thread operations. */
 enum class thread_operation { INSERT, UPDATE, READ, REMOVE, CHECKPOINT, TIMESTAMP, MONITOR };
+/* Define a thread that can be used to operate on collections */
```

Comment:
I think a better comment would be:
Container class for a thread and any data types it may need to interact with the database.